NEWS.md

```diff
@@ -1,3 +1,27 @@
+# protti 0.2.0.9000
+
+## New features
+
+* `fetch_pdb()` was added. It fetches PDB structure metadata from RCSB.
+* `fetch_pdb_structure()` was added. It fetches atom level data for a PDB structure from RCSB.
+* `fetch_metal_pdb()` was added. It fetches information about protein-metal binding sites from the MetalPDB database.
+* `find_peptide_in_structure()` was added. It returns the positions of a protein region, peptide or amino acid within a protein structure using UniProt positions as input. This is necessary because often amino acid positions in a protein strucutre vary from their positions in the UniProt protein sequence. 
+* `map_peptides_on_structure()` was added. It can visually map peptides onto PDB structures based on their positions. This is accomplished by replacing the B-factor information in the structure file with values that allow highlighting peptides when the structure is coloured by B-factor.
```

Comment:
Here I would not say that it can "visually map peptides" when in reality the visual part is done in pyMOL, Chimera, etc.